plutus-core/cost-model/create-cost-model/CreateBuiltinCostModel.hs

```diff
@@ -44,58 +44,60 @@ microToPico = ceiling . (1e6 *)
-  , paramChooseUnit               = "chooseUnitModel"
-  , paramTrace                    = "traceModel"
-  , paramFstPair                  = "fstPairModel"
-  , paramSndPair                  = "sndPairModel"
-  , paramChooseList               = "chooseListModel"
-  , paramMkCons                   = "mkConsModel"
-  , paramHeadList                 = "headListModel"
-  , paramTailList                 = "tailListModel"
-  , paramNullList                 = "nullListModel"
-  , paramChooseData               = "chooseDataModel"
-  , paramConstrData               = "constrDataModel"
-  , paramMapData                  = "mapDataModel"
-  , paramListData                 = "listDataModel"
-  , paramIData                    = "iDataModel"
-  , paramBData                    = "bDataModel"
-  , paramUnConstrData             = "unConstrDataModel"
-  , paramUnMapData                = "unMapDataModel"
-  , paramUnListData               = "unListDataModel"
-  , paramUnIData                  = "unIDataModel"
-  , paramUnBData                  = "unBDataModel"
-  , paramEqualsData               = "equalsDataModel"
-  , paramMkPairData               = "mkPairDataModel"
-  , paramMkNilData                = "mkNilDataModel"
-  , paramMkNilPairData            = "mkNilPairDataModel"
-  , paramSerialiseData            = "serialiseDataModel"
+  { paramAddInteger                      = "addIntegerModel"
+  , paramSubtractInteger                 = "subtractIntegerModel"
+  , paramMultiplyInteger                 = "multiplyIntegerModel"
+  , paramDivideInteger                   = "divideIntegerModel"
+  , paramQuotientInteger                 = "quotientIntegerModel"
+  , paramRemainderInteger                = "remainderIntegerModel"
+  , paramModInteger                      = "modIntegerModel"
+  , paramEqualsInteger                   = "equalsIntegerModel"
+  , paramLessThanInteger                 = "lessThanIntegerModel"
+  , paramLessThanEqualsInteger           = "lessThanEqualsIntegerModel"
+  , paramAppendByteString                = "appendByteStringModel"
+  , paramConsByteString                  = "consByteStringModel"
+  , paramSliceByteString                 = "sliceByteStringModel"
+  , paramLengthOfByteString              = "lengthOfByteStringModel"
+  , paramIndexByteString                 = "indexByteStringModel"
+  , paramEqualsByteString                = "equalsByteStringModel"
+  , paramLessThanByteString              = "lessThanByteStringModel"
+  , paramLessThanEqualsByteString        = "lessThanEqualsByteStringModel"
+  , paramSha2_256                        = "sha2_256Model"
+  , paramSha3_256                        = "sha3_256Model"
+  , paramBlake2b                         = "blake2bModel"
+  , paramVerifySignature                 = "verifySignatureModel"
+  , paramVerifyEcdsaSecp256k1Signature   = "verifyEcdsaSecp256k1SignatureModel"
+  , paramVerifySchnorrSecp256k1Signature = "verifySchnorrSecp256k1SignatureModel"
```

Comment:
The previous two entries (and similar ones below) are all that's been added here; it's caused a lot of realignment though.